src/App.js

```diff
@@ -1,14 +1,62 @@
 import React from 'react';
 import './App.css';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  NavLink,
+} from 'react-router-dom';
 import AddGroceries from './AddGroceries';
 import GroceryList from './GroceryList';
 
 function App() {
   return (
-    <div className="App">
-      <AddGroceries />
-      <GroceryList />
-    </div>
+    <Router>
+      <div className="App">
+        <h1>Shopping app</h1>
+        <Switch>
+          <Route path="/grocerylist">
+            <GroceryList />
+          </Route>
+          <Route path="/addgroceries">
+            <AddGroceries />
+          </Route>
```

Comment:
I would check with the other group to see if this works for them, demo-wise to have the input and list on different pages. 

If you are doing this, will you still need the `List` and `AddItem` components?

I might keep the paths more generic like `/list` and `/additem` just in case it's a more generic "shopping" list, rather than groceries-specific. Then maybe you could put your `List` and `AddItem` components back where you had them, and put the `GroceryList` and `AddGroceries` components on one page for now.

Reply:
Removing the List/AddItem components plus keeping the generic paths totally makes sense. We'll update to reflect that.
Regarding the input and list being on different pages, we thought one of the acceptance criteria* asked for that; we could have understood it wrong though. I agree with the fact that it doesn't look right though. 

*When one of the links is clicked, the browser URL updates to represent the current view and a matching view component is displayed

Reply:
Cool, yep, I see what you're saying. Good call. Just make sure the other team knows, I think. Nice work! 💯

Reply:
My understanding also was that the requirements called for separate views. In [this example](https://www.youtube.com/watch?v=mwj74mE9s64), there are separate views for the list and for adding items. Is this what we are going after? 

I'm also not sure how to implement all the components together (`List`, `AddItem`, `GroceryList` and `AddGroceries`). Should we have `GroceryList` and `AddGroceries` at '/', `List` at '/list' and `AddItem` at '/additem'?

Reply:
@cafloyd, @ablizben & @dariatsvetkova, Lauren brought up a great point regarding the UI of our app so far. You'll notice that because of the criteria I mentioned above, the AddGroceries and AddGroceryList aren't both displaying at the same time. Our React router only displays the corresponding component based on the link that's been clicked on. For apps like this, I've mainly seen this sort of behavior on mobile screen sizes, so it definitely looks a bit off.

Reply:
> My understanding also was that the requirements called for separate views. In [this example](https://www.youtube.com/watch?v=mwj74mE9s64), there are separate views for the list and for adding items. Is this what we are going after?
> 
> I'm also not sure how to implement all the components together (`List`, `AddItem`, `GroceryList` and `AddGroceries`). Should we have `GroceryList` and `AddGroceries` at '/', `List` at '/list' and `AddItem` at '/additem'?

@dariatsvetkova Yep, you are right. I was thinking of it as the "connecting to firestore" was a preliminary effort that might be completely re-written in future issues, but the way it's set up you could totally build off of it. I think the way you have it is 100% spot-on.

Reply:
@laurenmbeatty got it. I've removed the old components and updated the paths to keep them generic. 

To @kkatoch89's point, I agree that the way the UI is set up is a bit awkward on the desktop. Maybe we should brainstorm a better version as a team and open a new issue to implement it later on?

Reply:
Yeah, feel free to create issues on the board. I think this looks amaaaazing and you all rocked it. Very clean, and I'm good with merging :💯 %

Reply:
@laurenmbeatty thanks :) Should we get approval from all the mentors before merging?

Reply:
Nope, no need!  GO for it @dariatsvetkova